action/protocol/execution/protocol.go

```diff
@@ -75,15 +75,12 @@ func (p *Protocol) Handle(ctx context.Context, act action.Action, sm protocol.St
 	return receipt, nil
 }
 
-// Validate validates an execution
+// Validate validates the size of an execution
```

Comment:
revert this, current code only checks size, but `Validate` is meant to validate entire action